### ISSUE TITLE
Tasks without section sometimes still display a section on the `list` command

### ISSUE THREAD

**Issue by andreoliwa** [sachaos/todoist#161]
Hi!

There are tasks without a section that still display a section when I use the `list` command.

This bug doesn't happen with all tasks.
Steps to reproduce:

1. Create a new project called "Example".
2. Add a task called "Some task" without a section.
3. Run `todoist sync && todoist list -f '#example'`: the project shows as `#Example`. ✅ 
3. Create a new section called "My Section".
4. Move "Some task" to "My Section".
5. Run `todoist sync && todoist list -f '#example'`: the project shows as `#Example/My Section`. ✅ 
6. Move "Some task" out of "My Section", back to the "(No Section)" area.
7. Run `todoist sync && todoist list -f '#example'`: the project still shows as `#Example/My Section`. ❌ 
8. The correct output is displayed with the `show` command.

In the details below: a (clean) transcript of my terminal output and some screenshots.

<details>

```shell
$ todoist sync && todoist list -f '#example'

$ todoist sync && todoist list -f '#example'
4693482710 p4  #Example  Some task

$ todoist sync && todoist list -f '#example'
4693482710 p4  #Example/My Section  Some task

$ todoist sync && todoist list -f '#example'
4693482710 p4  #Example/My Section  Some task

$ todoist sync && todoist show 4693482710
ID       4693482710
Content  Some task
Project  #Example
Labels
Priority p4
DueDate
URL
```

![Screenshot 2021-03-26 at 19 55 12](https://user-images.githubusercontent.com/1258218/112680112-40857680-8e6d-11eb-9c0f-4f39bd1d2707.png)
![Screenshot 2021-03-26 at 19 55 02](https://user-images.githubusercontent.com/1258218/112680118-41b6a380-8e6d-11eb-8a3f-21eb26ef37e4.png)
![Screenshot 2021-03-26 at 19 54 47](https://user-images.githubusercontent.com/1258218/112680120-424f3a00-8e6d-11eb-82d9-5aa29ee4c4d9.png)

</details>

**Comment by andreoliwa** [comment]
I found out that removing the local cache is a temporary solution for this problem:

```
$ rm -rvf ~/.cache/todoist
/Users/aa/.cache/todoist/cache.json
/Users/aa/.cache/todoist
```

So maybe it's something related to loading or saving empty/null sections in the cache. 🤔

**Comment by dwrdx** [comment]
Yes, you are right about that, It's bug happened  when there are null sections. Pull request  #162 is to solve this issue.
